gettsim/parameters/soz_vers_beitr.yaml

```diff
@@ -91,71 +100,65 @@ soz_vers_beitr:
       standard: 0.005
   1997-01-01:
     ges_krankenv:
-      ag: 0.067
-      an: 0.067
+      allg: null
+      erm: null
+      zus: 0
     ges_pflegev:
       standard: 0.0085
       zusatz_kinderlos: 0
     arbeitsl_v: 0.0325
     ges_rentenv: 0.1015
-  1998-01-01:
-    deviation_from: previous
-    ges_krankenv:
-      ag: 0.068
-      an: 0.068
   2000-01-01:
     deviation_from: previous
     ges_krankenv:
-      ag: 0.06785
-      an: 0.06785
+      allg: null
     ges_rentenv: 0.0965
   2001-01-01:
     deviation_from: previous
     ges_krankenv:
-      ag: 0.0679
-      an: 0.0679
+      allg: null
     ges_rentenv: 0.0955
   2002-01-01:
     deviation_from: previous
     ges_krankenv:
-      ag: 0.07
-      an: 0.07
+      allg: null
   2003-01-01:
     deviation_from: previous
     ges_krankenv:
-      ag: 0.072
-      an: 0.072
+      allg: null
     ges_rentenv: 0.0975
   2004-01-01:
     deviation_from: previous
     ges_krankenv:
-      ag: 0.071
-      an: 0.071
+      allg: null
   2005-01-01:
     ges_krankenv:
-      ag: 0.0689
-      an: 0.0689
+      allg: null
+      zus: 0
+      erm: null
     ges_pflegev:
       standard: 0.0085
       zusatz_kinderlos: 0.0025
     arbeitsl_v: 0.0325
     ges_rentenv: 0.0975
-  2006-01-01:
+  2005-07-01:
     deviation_from: previous
     ges_krankenv:
-      ag: 0.0666
-      an: 0.0756
+      allg: null
+      zus: 0.009
+    note: Allgemeiner Beitragssatz wird nicht mehr gleichmäßig aufgeteilt, Einführung eines Zusatzbeitrags
```

Comment:
Adjust note. If I understand correctly, this is also the first time that general contribution rates are defined (and part of the GETTSIM parameters)

Reply:
No, the general ones were introduced in 2009.